Source/SvgDocument.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
It seems that the file encoding has changed.
Please apply setting of EditorConfig.

Reply:
According to VS code, the file is encoded in UTF-8 with BOM and CRLF, as is specified in .editorconfig. Also, I compared the previous and new version of this file with a HEX editor and the first couple of rows are identical. To be honest, I have no idea why GitHub highlights the first line as a change.

Reply:
BOM does not exist.

Reply:
It should now be present